### PR TITLE
Tweaks to running instructions and show how to use reactor-netty snapshot builds

### PR DESCRIPTION
**Pull request by crankydillo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,23 +1,45 @@
...
 
-# How to build 
-`mvn clean install` 
+This is the first thing you need to do.
 
-## IDE
-Run MockService 
-You could use mvn exec:java -pl mockapplication too.
+```sh
+mvn clean install
```
Comment (sebady) [inline README.md:9]: Should we specify `-pl benchmarkcommon` here?

**Review by sebady — APPROVED** [review]
LGTM

File: reactornettybenchmark/pom.xml [hunk 2]
```diff
@@ -15,8 +15,25 @@
     <properties>
         <reactornetty.version>0.8.3.RELEASE</reactornetty.version>
         <reactor.version>3.2.2.RELEASE</reactor.version>
+        <!--
+        <reactornetty.version>0.8.4.BUILD-SNAPSHOT</reactornetty.version>
```
Comment (sebady) [inline reactornettybenchmark/pom.xml:19]: Should we make the snapshot the default and comment out release version?

File: README.md (hunk 1 above)
Comment (crankydillo) [reply README.md:9]: I want to if we're removing the `performance` profile.  If not, I don't see much harm in leaving it as is.  Are you onboard with removing the profile?

**Review by sharathsrinivasa — APPROVED** [review]
LGTM

File: reactornettybenchmark/pom.xml (hunk 2 above)
Comment (sharathsrinivasa) [reply reactornettybenchmark/pom.xml:19]: If you OK with dealing with instability with SNAPSHOT version. IMO, lets keep this as stable version and toggle to SNAPSHOT if (rxnetty issues claims certain issues are fixed) there is a need.

File: README.md (hunk 1 above)
Comment (sebady) [reply README.md:9]: I would vote for the profile removal if there is not additional complexity in test setup/execution

**Pull request merged**